core/api.go

```diff
@@ -99,6 +99,7 @@ func (h *Handler) init() {
 	h.actionDecoders = map[string]func(data []byte) (txbuilder.Action, error){
 		"control_account":                h.Accounts.DecodeControlAction,
 		"control_program":                txbuilder.DecodeControlProgramAction,
+		"control_with_receiver":          txbuilder.DecodeControlWithReceiverAction,
```

Comment:
Yeah, I think the wire protocol and the SDK interface are very different domains. I don't think the wire protocol should be considered part of the product/public interface. It's an implementation detail.

There's still value in consistency within the wire protocol for the same reason we value consistency in code style. Consistency between the SDKs and wire protocol is nice if it's easy, but otherwise, not worth it IMO.